hacl-rs/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Utility modules. In the generated hacl-rs, these are individual crates.
```

Comment:
We should add a warning into the docs or somewhere else, that this crate is only for internal use.